app/models/user.rb

```diff
@@ -4,12 +4,14 @@ class User < ApplicationRecord
   has_many :chirps, foreign_key: :author_id, inverse_of: :author
   has_many :follows, foreign_key: :follower_id, inverse_of: :follower
   has_many :likes, foreign_key: :liker_id, inverse_of: :liker
+  has_many :reactions, foreign_key: :reacter_id, inverse_of: :reacter
   has_many :incoming_follows, class_name: "Follow", foreign_key: :friend_id, inverse_of: :friend
   has_and_belongs_to_many :mentions, class_name: "Chirp"
 
   has_many :friends, through: :follows
   has_many :liked_chirps, through: :likes, source: :chirp
   has_many :followers, through: :incoming_follows
+  has_many :reacted_chirps, through: :reactions, source: :chirp
```

Comment:
@jasmine-q I'm not super familiar with this code, does this need to be passed through to the controller too?